tests/libstorage/storageclass.go

```diff
@@ -253,6 +253,19 @@ func GetBlockStorageClass(accessMode k8sv1.PersistentVolumeAccessMode) (string,
 	return sc, foundSC
 }
 
+// GetAvailableRWFileSystemStorageClass returns any RWX or RWO access mode filesystem storage class available, i.e,
+// If the available filesystem storage classes only support RWO access mode, it returns that SC or vice versa.
+// This method to get a filesystem storage class is recommended when the access mode is not relevant for the purpose of
+// the test.
+func GetAvailableRWFileSystemStorageClass() (string, bool) {
```

Comment:
Maybe use named return values here to document what the `bool` does return.

Reply:
I'll take that back - the other funcs also don't comment, so rather leave it like this.